### DISCUSSION TITLE
欢迎来到BusMonitor_Wuhan 讨论!

### DISCUSSION THREAD

**Discussion by a2633063** [a2633063/BusMonitor_Wuhan#4] [General]
## 👋 欢迎
 就先把这里当做讨论的地方吧.
你可以在这里问你想的问题,分享你的想法,与其他用户接触!
你可以在这里做任何你想做且不会让别人反感的任何事情.(当然违法违规的不行!)

**Comment by 2311450936** [comment]
接口是不是没了，whgjzt的
